compiler/header.h

```diff
@@ -388,7 +386,7 @@ struct options {
     FILE * output_h;
     byte syntax_tree;
     byte comments;
-    byte js_esm;
+    bool js_esm;
```

Comment:
Perhaps we should consider moving over to using C's `bool` type (we do now require a C99 compiler to build the compiler) but I don't think it's helpful to change a single place to use `bool` like this.

Reply:
```suggestion
    byte js_esm;
```